apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -184,9 +172,9 @@ If you want to learn more about Prisma, check out the [docs](https://www.prisma.
 
 ## Troubleshooting
 
-### 1. Missing grants
+### Missing grants
 
-If you run `prisma migrate dev --name init` multiple times, it sometimes asks if you want to recreate the whole schema. If you chose yes, it will delete the public schema and recreate it. The default grants are missing after this.
+If you run `prisma migrate dev --name init` multiple times, it sometimes asks if you want to reset your database schema. If you chose yes, it will delete the public schema and recreate it. The default grants are missing after this.
```

Comment:
This is still incorrect. You can run `migrate dev` a million times and it will not ask your to reset anything.

Reply:
Updated. Let me know if that reads better